### ISSUE TITLE
getting "cannot read data from private feild" erro in my grafana integrated with cloudwatch

### ISSUE THREAD

**Issue by sajidkhan8530** [grafana/grafana#90863]
**Steps to reproduce**:

**Actual Result**:

**Expected Result**

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)

**Environment**:
- Grafana version:
- Data source type & version:
- User OS & Browser:
- Others:

**Comment by tonypowa** [comment]
Hi @sajidkhan8530 

Thank you for opening this issue

To assist you better, could you please provide some additional details that would help us reproduce the problem on our end? This will allow us to investigate and resolve the issue more efficiently.

Any specifics you can share would be greatly appreciated.

**Comment by tonypowa** [comment]
I’ll be closing this issue due to inactivity, but please feel free to request a reopening if you have any additional questions or concerns
